data/Parmesh119.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "Parmesh Bhatt",
+  "type": "personal",
+  "bio": "#119 | Junior IT'25 Engineer👨‍🎓 || Developer Program Member @GitHub || Cloud Team Member @CLDC || 1X @AWS certified || DSA💡|| Open-Source🖥 in Cloud computing☁ and DevOps || Connecting with Global-Communities 🌍 |"
+  "avatar": "https://github.com/Parmesh119.png",
```

Comment:
Please add the missing comma `,` in the end of line no. `4` and remove the `avatar` key from your json as we no longer need it. 

```suggestion
  "bio": "#119 | Junior IT'25 Engineer👨‍🎓 || Developer Program Member @GitHub || Cloud Team Member @CLDC || 1X @AWS certified || DSA💡|| Open-Source🖥 in Cloud computing☁ and DevOps || Connecting with Global-Communities 🌍 |",
```